src/include/optimizer/absexpr_expression.h

```diff
@@ -0,0 +1,176 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// absexpr_expression.h
+//
+// Identification: src/include/optimizer/absexpr_expression.h
```

Comment:
In general, if I understand correctly, it is more convenient for you to be operating on something that is not directly the parser's expressions. We are bringing in logical operators or something like that (PR 232) in terrier, which are the outputs of the binder and inputs to the optimizer. This seems like it might belong there, then.